book/nushell_map_imperative.md

```diff
@@ -24,37 +24,37 @@ Note: this table assumes Nu 0.43 or later.
+| mkdir        | os.mkdir                      |                                                     |                         | fs::create_dir                                |
+| mv           | shutil.move                   |                                                     |                         | fs::rename                                    |
 | get          | list[x]                       | List[x]                                             | vector[x]               | Vec[x]                                        |
 | open         | open                          |                                                     |                         |                                               |
 | transpose    | zip(\*matrix)                 |                                                     |                         |                                               |
 | http post    | urllib.request.urlopen        |                                                     |                         |                                               |
 | prepend      | deque.appendleft              |                                                     |                         |                                               |
 | print        | print                         | println                                             | printf                  | println!                                      |
 | ps           | os.listdir('/proc')           |                                                     |                         |                                               |
-| pwd          | os.getcwd                     |                                                     |                         |                                               |
+| pwd          | os.getcwd                     |                                                     |                         | env::current_dir                              |
 | range        | range                         | .., until, downTo, step                             | iota                    | ..                                            |
 | reduce       | functools.reduce              | reduce                                              | reduce                  | fold, rfold, scan                             |
 | reject       | del                           |                                                     |                         |                                               |
-| rename       | shutil.move                   |                                                     |                         |                                               |
+| rename       | shutil.move                   |                                                     |                         | fs::rename                                    |
```

Comment:
This is a mistake on the python side already. `rename` doesn't affect the file system but replaces the keys of a table/records

Reply:
oh right, yeah, i just looked at the python command. i'll change both now, should I rename keys in hashmaps for both?